tools/internal_ci/linux/grpc_build_submodule_at_head.sh

```diff
@@ -20,6 +20,7 @@ set -ex
 # change to grpc repo root
 cd $(dirname $0)/../../..
 
+export PREPARE_BUILD_INSTALL_DEPS_PYTHON=true
```

Comment:
as expressed in my other comment, this has actually nothing to do with "python dependencies" and it ends up being an arbitrary hack.